dashboard/test/controllers/lessons_controller_test.rb

```diff
@@ -189,6 +189,50 @@ class LessonsControllerTest < ActionController::TestCase
     assert_includes @response.body, section.name
   end
 
+  test 'can not show student lesson plan when lesson is in a non-migrated script' do
```

Comment:
Great tests check that we can't show the page in these two cases. We probably want one test to show that we can view the student lesson plan when people are supposed to